config.py

```diff
@@ -1522,11 +1539,14 @@ def get_datatype() -> Literal['meg', 'eeg']:
 def _get_reject(
     reject: Optional[Dict[str, float]],
     ch_types: Iterable[Literal['meg', 'mag', 'grad', 'eeg']]
-) -> Dict[str, float]:
+) -> Union[Dict[str, float], Literal['auto']]:
     if reject is None:
         return dict()
 
-    reject = reject.copy()
+    if reject == 'auto' :
+        return 'auto'
+
+    reject_ = reject.copy()  # Avoid clash with global variable.
```

Comment:
This does a shallow copy right?  Another good way of copying this dict is `dict(reject)`.